modules/osm/tags.js

```diff
@@ -176,6 +178,23 @@ export var osmOneWayTags = {
         'tidal_channel': true
     }
 };
+export const osmOneWayBackwardTags = {
+    'conveying': {
+        'backward': true,
+    },
+    'oneway': {
+        '-1': true,
+    },
+};
+export const osmBiDirectionalTags = {
```

Comment:
Was there a particular reason to deviate here from the naming scheme of the other two related obejcts? 

```js
export const osmOneWayBiDirectionalTags = {
```

Technically, `osmBiDirectionalTags` could be confused with the concept of "regular non-oneway" features?!

Reply:
no real reason, changed now